.github/workflows/go-mod-sync.yaml

```diff
@@ -0,0 +1,41 @@
+name: Meshery extensions go.mod sync
```

Comment:
@Revolyssup, perhaps, this workflow should be just another step in the existing Edge and Stable release workflows?

Reply:
This only gets triggered when the go.mod changes. The rest are general.

Reply:
OK. 👍